src/js/utils/events.js

```diff
@@ -41,6 +41,22 @@ export class Observable {
 
 export const observable = new Observable();
 
+export function observeStore(store, select, onChange) {
+    let currentState;
+
+    function handleChange() {
+        const nextState = select(store.getState());
+        if (nextState !== currentState) {
```

Comment:
this function is generic, the `select` param is a function that specify what is interesting in the store (and if you look at https://github.com/smooch/smooch-js/pull/223/files#diff-b089bf546efa30553b8f38a397c6c748R144, I'm only interested in the conversation key)